client/my-sites/posts/post.jsx

```diff
@@ -333,6 +333,8 @@ const Post = React.createClass( {
 					<Comments
 						showCommentCount={ false }
 						post={ this.props.post }
+						showFilters
+						showModerationTools
```

Comment:
Would this be a good place to put: `showModerationTools={ config.isEnabled( ... ) }`?